packages/cxl-ui/src/components/cxl-details/cxl-details.js

```diff
@@ -0,0 +1,17 @@
+import { customElement } from 'lit/decorators.js';
```

Comment:
I don't get it, why is this intermediate component necessary at all?

Reply:
It most likely isn't. Originally I wanted to see if I could get a collapsible element with similar styles to the current checkout accordion. I'm realizing this isn't an easy feat. The styles for the checkout elements start here: https://github.com/conversionxl/institute-theme/blob/institute/scss/_50-woocommerce.scss
 though there are some nested dependencies.